### PR TITLE
Add no-mouse-days post

### PR DESCRIPTION
**Pull request by marcysutton**
Prompted by @ericwbailey, this PR has a contribution post about my No Mouse Days package and its origins. I'd love any and all feedback!

### CODE REVIEW

File: src/posts/2020-10-15-no-mouse-days.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    source: Marcy Sutton
+    year: 2019
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - howto
+---
+
+Wouldn’t it be great if designers and developers tested their digital experiences for keyboard accessibility–and by extension, screen reader accessibility–on the regular? That’s where the #NoMouse challenge and No Mouse Days come in.
```
Comment (ericwbailey) [inline src/posts/2020-10-15-no-mouse-days.md:27]: I love me a good emdash, but could we break this into two sentences? I'd like to try and keep the reading level lower.

File: src/posts/2020-10-15-no-mouse-days.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+The DO-IT Center at University of Washington originally started the [No Mouse challenge](https://nomouse.org/), complete with its own website, logo, and [hashtag](https://twitter.com/hashtag/nomouse) to spread awareness about the need for keyboard-accessible websites and applications.
+
+The challenge is pretty straight-forward:
```
Comment (ericwbailey) [inline src/posts/2020-10-15-no-mouse-days.md:31]:
```suggestion
The challenge is pretty straight-forward:

```

File: src/posts/2020-10-15-no-mouse-days.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+1. **Step 1. Use the Web without a mouse.**
+  Can you access all features? Can you reach and operate all interactive controls? Can you easily tell where you are on the page?
+2. **Step 2. Learn more about accessible web design.**
+3. **Step 3. Spread the Word!**
+
+Building on that initiative came [No Mouse Days](https://npmjs.com/package/no-mouse-days), an npm package that disables the mouse cursor on web pages with CSS. It started as a cheeky joke, and then it actually became [popular](https://twitter.com/MiriSuzanne/status/1184142272968953856).
+
+The idea of No Mouse Days was to make it easy to hide the mouse cursor so it would become extremely obvious when a user interface didn’t support the keyboard, perhaps disabling it automatically for front-end development one day a week. True to its origins it’s not the most robust package, but people have actually found it useful. If you have further ideas for No Mouse Days or other culture-changing techniques, contributions are always welcome on [GitHub](https://github.com/marcysutton/no-mouse-days) and [Twitter](https://twitter.com/marcysutton)!
```
Comment (ericwbailey) [inline src/posts/2020-10-15-no-mouse-days.md:39]: >True to its origins it’s not the most robust package, but people have actually found it useful.

What are your thoughts on removing this sentence? My thinking here is its a great little package that does a simple thing to communicate a powerful idea. I don't think you need to disparage yourself!

File: src/posts/2020-10-15-no-mouse-days.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+No matter how you go about it, establishing a mouse-free development day at least once a week could work wonders for users who can’t use a mouse at all. You might find interactive controls that aren’t actually interactive for large segments of the population and lacking visible focus outlines which make keyboard navigation difficult.
```
Comment (ericwbailey) [inline src/posts/2020-10-15-no-mouse-days.md:41]:
```suggestion
No matter how you go about it, establishing a mouse-free development day at least once a week could work wonders for people who can’t use a mouse at all. You might find interactive controls that aren’t actually interactive for large segments of the population and lacking visible focus outlines which make keyboard navigation difficult. 
```

[House style](https://www.a11yproject.com/content-style-guide/#user-or-person-) 😁

File: src/posts/2020-10-15-no-mouse-days.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - howto
+---
+
+Wouldn’t it be great if designers and developers tested their digital experiences for keyboard accessibility–and by extension, screen reader accessibility–on the regular? That’s where the #NoMouse challenge and No Mouse Days come in.
+
+The DO-IT Center at University of Washington originally started the [No Mouse challenge](https://nomouse.org/), complete with its own website, logo, and [hashtag](https://twitter.com/hashtag/nomouse) to spread awareness about the need for keyboard-accessible websites and applications.
```
Comment (ericwbailey) [inline src/posts/2020-10-15-no-mouse-days.md:29]: TIL!

**Review by ericwbailey — CHANGES_REQUESTED** [review]
Love it. A few small things, and then I think this is ready for publication!

File: src/posts/2020-10-15-no-mouse-days.md (hunk 1 above)
Comment (marcysutton) [reply src/posts/2020-10-15-no-mouse-days.md:27]: Sure thing!

File: src/posts/2020-10-15-no-mouse-days.md (hunk 3 above)
Comment (marcysutton) [reply src/posts/2020-10-15-no-mouse-days.md:39]: Works for me!

**Comment by marcysutton** [comment]
I've made the updates, so this PR should be good to go! Thanks for the review.

**Review by ericwbailey — APPROVED** [review]

**Comment by ericwbailey** [comment]
Thank you so much! We'll announce the post tomorrow on Twitter and have it linked out in our newsletter. What's the best way to send you payment? You can email or DM me if you're not comfortable sharing that info here.

**Pull request merged**